src/content/tutorials/observability/multi-tenancy/_index.md

```diff
@@ -1,15 +1,201 @@
 ---
 title: Multi-tenancy
-description: Guides explaining how to configure multi-tenancy in the Observability Platform.
+description: Understanding multi-tenancy concepts and best practices for the Observability Platform.
 weight: 20
 menu:
   principal:
     parent: tutorials-observability
     identifier: tutorials-observability-multitenancy
-last_review_date: 2024-12-12
+last_review_date: 2025-06-11
 owner:
   - https://github.com/orgs/giantswarm/teams/team-atlas
 user_questions:
-  - How to isolate data into the observability platform?
-  - How to create grafana organizations?
+  - How to isolate data in the observability platform?
+  - What are tenant separation best practices?
+  - How do tenants and organizations work together?
 ---
+
+The observability platform supports multi-tenancy to help you isolate data and control access across different teams, environments, or projects. This guide explains the core concepts and best practices for implementing tenant separation.
+
+## Understanding multi-tenancy
+
+Multi-tenancy works through three key components:
+
+### Tenants
+
+A **tenant** is a logical namespace that isolates observability data (metrics, logs) in the backend storage systems (Mimir and Loki). When applications send data, they include a tenant label that determines where the information goes.
+
+**Key characteristics:**
+
+- Complete data isolation at the storage level
+- Independent retention policies and limits
+- Separate query scope for dashboards and alerts
+
+### Grafana organizations
+
+A **Grafana organization** gives you access to specific tenants through dedicated datasources and dashboards. Each organization is a separate workspace in Grafana with its own:
+
+- User roles and permissions
+- Dashboards and folders
+- Datasource configurations
+- Alert rules and notification channels
```

Comment:
Alert rules and alertmanager config are more related to tenants than organizations I think?

Reply:
Makes sense. Do you mind providing a suggestion for this?

Reply:
```suggestion
```